### PR TITLE
EYYY this is my pull request

### PR DESCRIPTION
**Pull request by idetore**
To whom it may concern, 

This is my pull request. Thanks!

Best,
Ian

### CODE REVIEW

**Comment by TG-Techie** [comment]
Ian, please describe what changes you made to the branch in the above comment and the name of the pull request. We trust you made the changes  correctly but would like to know what they are.

File: firmware/robot/GPSWrapper.cpp [hunk 1]
```diff
@@ -65,7 +66,7 @@ String GPSWrapper::getAltitude(){//If there is a fix, returns the altitude in me
     else{return String(GPS.altitude,1)+String(" meters");}
 }
 
-String GPSWrapper::getTime(){                   //returns the time in “HH:MM:SS.SSS UTC” Time is UTC, not timezone or daylight saving offset.
+String GPSWrapper::getTime(){                   //returns the time in â€œHH:MM:SS.SSS UTCâ€� Time is UTC, not timezone or daylight saving offset.
```
Comment (JeremyParadie) [inline firmware/robot/GPSWrapper.cpp:69]:
```suggestion
String GPSWrapper::getTime(){                   //returns the time in "HH:MM:SS.SSS UTC" Time is UTC, not timezone or daylight saving offset.
```

File: firmware/robot/GPSWrapper.cpp [hunk 2]
```diff
@@ -84,9 +85,9 @@ String GPSWrapper::getTime(){                   //returns the time in “HH:MM:S
   return Time;                                  //return formatted time
 }
 
-String GPSWrapper::getDate(){                   //returns date in “YYYY-MM-DD” format
+String GPSWrapper::getDate(){                   //returns date in â€œYYYY-MM-DDâ€� format
```
Comment (JeremyParadie) [inline firmware/robot/GPSWrapper.cpp:88]:
```suggestion
String GPSWrapper::getDate(){                   //returns date in "YYYY-MM-DD" format
```

File: firmware/robot/GPSWrapper.cpp [hunk 3]
```diff
@@ -84,9 +85,9 @@ String GPSWrapper::getTime(){                   //returns the time in “HH:MM:S
...
   parse();//looks for new sentence from the GPS and parses it
-  return String("20")+String(GPS.year)+String("-")+String(GPS.month)+String("-")+String(GPS.day); //return date in “YYYY-MM-DD” format
+  return String("20")+String(GPS.year)+String("-")+String(GPS.month)+String("-")+String(GPS.day); //return date in â€œYYYY-MM-DDâ€� format
```
Comment (JeremyParadie) [inline firmware/robot/GPSWrapper.cpp:90]:
```suggestion
  return String("20")+String(GPS.year)+String("-")+String(GPS.month)+String("-")+String(GPS.day); //return date in "YYYY-MM-DD" format
```

File: firmware/robot/GPSWrapper.h [hunk 4]
```diff
@@ -21,10 +21,11 @@ class GPSWrapper{                     //Wrapper class for the GPS library- this
     void read();                      //Reads in a new char from GPS, if there is one. this needs to be called very often, around 1khz (a thousand times per second)
     String getLocation();             //If there is a fix, returns the location in decimal degrees lattitude and longitude. if there isnt a fix, returns "Waiting for fix"
     String getAltitude();             //If there is a fix, returns the altitude in meters. example: "76.5 meters" if there isnt a fix, returns "Waiting for fix"
-    String getTime();                 //Returns the time in “HH:MM:SS.SSS UTC” Time is UTC, not timezone or daylight saving offset.
-    String getDate();                 //Returns date in “YYYY-MM-DD” format
+    String getTime();                 //Returns the time in â€œHH:MM:SS.SSS UTCâ€� Time is UTC, not timezone or daylight saving offset.
```
Comment (JeremyParadie) [inline firmware/robot/GPSWrapper.h:24]:
```suggestion
    String getTime();                 //Returns the time in "HH:MM:SS.SSS UTC" Time is UTC, not timezone or daylight saving offset.
```

Comment (JeremyParadie) [inline firmware/robot/GPSWrapper.h:24]:
```suggestion
    String getTime();                 //Returns the time in "HH:MM:SS.SSS UTC" Time is UTC, not timezone or daylight saving offset.
```

File: firmware/robot/GPSWrapper.h [hunk 5]
```diff
@@ -21,10 +21,11 @@ class GPSWrapper{                     //Wrapper class for the GPS library- this
...
+    String getDate();                 //Returns date in â€œYYYY-MM-DDâ€� format
```
Comment (JeremyParadie) [inline firmware/robot/GPSWrapper.h:25]:
```suggestion
    String getDate();                 //Returns date in "YYYY-MM-DD" format
```

**Review by JeremyParadie — CHANGES_REQUESTED** [review]
please fix the encoding issue with the double quotes and the reformatting of the parse function in robot.ino

**Pull request closed without merging**